index.js

```diff
@@ -116,12 +116,93 @@ LocationForecast.prototype = {
       if (err) {
         return callback(err, null);
       }
-      
+
       return callback(null, res);
     });
   },
 
 
+    /**
+     * Get seven day weather.
+     * @param {Function} callback
+     */
+    getSevenDaySummary: function(callback) {
+      var self = this;
+      var fns = [];
+      // Use miday for forecast
+      var curDate = moment.utc().hours(12);
+      curDate.set('minutes', 0);
+
+      // Create 7 days of dates
+      for (var i = 0; i < 7; i++) {
+        // Avoid scope error for wrong day reference
+        (function() {
+          var day = curDate.clone();
+          day = day.add('days', i);
+          fns.push(function(cb) {
+            self.getForecastForTime(day, function(err, weather) {
+              if (err) {
+                return cb(err, null);
+              }
+
+              return cb(null, weather);
+            })
+          });
+        })()
```

Comment:
I'm not 100% sure on this, but might be a good idea to pass `i` here, and then on line 139 reference it as `day` to avoid possible reference issues? Your existing closure might already deal with this.